python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -3,14 +3,23 @@
 from pathlib import Path
 from typing import Optional, Sequence, Union
 
+import yaml
 from dagster._annotations import experimental
 from dagster._model import DagsterModel
 from dagster._utils import run_with_concurrent_update_guard
 
-from .errors import DagsterDbtManifestNotFoundError, DagsterDbtProjectNotFoundError
+from .errors import (
+    DagsterDbtManifestNotFoundError,
+    DagsterDbtProjectNotFoundError,
+    DagsterDbtProjectYmlFileNotFoundError,
+)
 
 logger = logging.getLogger("dagster-dbt.artifacts")
 
+DBT_MANIFEST_PREPARATION_IN_PROGRESS = "DBT_MANIFEST_PREPARATION_IN_PROGRESS"
+
+DBT_MANIFEST_PREPARATION_LOCK = "dbt_manifest_preparation.lock"
```

Comment:
rm

Reply:
Done in [993630c](https://github.com/dagster-io/dagster/pull/21296/commits/993630c7c19aa3dd3b59403734620db2b05ec271)